README.md

```diff
@@ -58,21 +60,20 @@ We use Ruby's built-in logger to output messages to standard output. Learn more
 
 Here's an example of output from the courses scraper:
 
-```
+```log
 [2018-10-18 01:35:01] INFO  (courses_scraper): Searching for courses in term 201801
 [2018-10-18 01:35:02] INFO  (courses_scraper): 178 department/semesters so far
 [2018-10-18 01:35:02] INFO  (courses_scraper): Searching for courses in term 201805
 [2018-10-18 01:35:03] INFO  (courses_scraper): 301 department/semesters so far
 ```
 
-The formatting for outputted messages is as follows:`[DATE TIME] LOG_LEVEL (PROGRAM_NAME): {MESSAGE}`
+The formatting for outputted messages is as follows: `[DATE TIME] LOG_LEVEL (PROGRAM_NAME): {MESSAGE}`
 
-An example of a log call in ruby:
-`logger.info(prog_name) {"MESSAGE"}`
```

Comment:
if you're gonna explicityly say this is a ruby example and put it on a new line, why not do something like
```ruby
logger.info(prog_name) {"MESSAGE"}
```
?

Reply:
w8 this is old code, but I think (?) it could still apply?

Reply:
Didn't actually touch this